### DISCUSSION TITLE
Is there a way to know the absolute pixel position of an object inside the score container?

### DISCUSSION THREAD

**Discussion by fablau** [opensheetmusicdisplay/opensheetmusicdisplay#1363] [Q&A]
By reading ["Exploring the Demo"](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/wiki/Exploring-the-Demo) I see there are easy ways to know the absolute and relative position of elements inside the OSMD positioning system, but I see no mentioned way to know the absolute position in pixels of an object inside the score container (either that's a DIV or any other HTML element). Is that something missing in the features set right now, or am I just missing it? Or is there a workaround to calculate that?

Thanks in advance for any help.

All the best,

**Comment by sschmidTU** [comment]
It's essentially the absolute position * 10 (unless maybe zoom is used)

osmd.graphic.domToSvg({x: number, y: number})
osmd.graphic.svgToDom({x: number, y: number})

See here:
https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/issues/504

**Reply by fablau** [reply]
Thank you Simon, I now understand. ;)
